Kconfig

```diff
@@ -20,9 +20,19 @@ config HP_MEMORY_BANKS
 config LP_MEMORY_BANKS
 	int "LP memory banks count"
 	default 1
+	depends on CAVS
```

Comment:
`depends` duplication, if it's already defined for menu it's automatically mandatory for menu entries.